### PR TITLE
Update README.md with link to getting started for Julia projects

### LINKED ISSUE #312: The `EpiAware` `README.md` file should be updated to have a link to the getting started with Julia for projects (https://cdcgov.github.io/Rt-without-renewal/dev/man/getting-started-julia/) to replace the bit there already.

### PR DESCRIPTION
**Pull request by SamuelBrand1**
As per title.

Closes #312

### CODE REVIEW

File: EpiAware/README.md [hunk 1]
```diff
@@ -90,15 +90,8 @@ H-.->I
...
-- `Revise`: For modifying package code and using the changes without restarting Julia session.
-- `Term`: For pretty and stylized REPL output (including error messages).
-- `JuliaFormatter`: For code formatting.
-- `Documenter`: For local documentation generation (useful for testing).
-- `Pluto`: For interactive development and testing.
-- `TestEnv`: For test development.
+## Opinionated guide to using Julia for project development
+Some user and potential contributors may not be familiar with using Julia for, or part of, project development. In documentation we give our opinions on how to use Julia for project development focussing on Julia version control with the command line tool `juliaup`, typical patterns for using stacked environments and useful settings for development using the Julia extension of VS-Code as an interactive development environment (IDE) for Julia project. Please find the documentation [here](https://cdcgov.github.io/Rt-without-renewal/dev/man/getting-started-julia/).
 
 ### `startup.jl` recommendation
```
Comment (seabbs) [inline EpiAware/README.md:96]: These should also really be in the Julia guide?

**Review by seabbs — APPROVED** [review]
Looks good. One comment that is a potential new issue.

File: EpiAware/README.md (hunk 1 above)
Comment (SamuelBrand1) [reply EpiAware/README.md:96]: These comments or the deleted list? The list is now in docs?

https://cdcgov.github.io/Rt-without-renewal/dev/man/getting-started-julia/#Recommended-packages-for-the-primary-Julia-environment

Comment (seabbs) [reply EpiAware/README.md:96]: the information below about `startup.jl`

**Pull request merged**